File: week3/student.cpp
#include <iostream>
using namespace std;
class Student
{ public: string fullName; 
int rollNum; 
double semPerentage; 
string collegeName; 
int collegeCode; 
Student()
 { 
 cout<<"Student class initialized!!\nIam a student\n";
 }
~Student(){} 
};
int main() 
{
Student LakshmiPriya;
cout<<"Enter your name : ";
getline(cin,LakshmiPriya.fullName);
cout<<"Enter your roll number : ";
cin>>LakshmiPriya.rollNum;
cout<<"Enter your sem percentage : ";
cin>>LakshmiPriya.semPerentage;
cout<<"Enter your college name : ";
cin>>LakshmiPriya.collegeName;
cout<<"Enter your college code : ";
cin>>LakshmiPriya.collegeCode;
cout<<"Name : "<<LakshmiPriya.fullName<<endl;
cout<<"RollNum : "<<LakshmiPriya.rollNum<<endl;
cout<<"SemPercentage : "<<LakshmiPriya.semPerentage<<endl;
cout<<"CollegeName : "<<LakshmiPriya.collegeName<<endl;
cout<<"CollegeCode : "<<LakshmiPriya.collegeCode<<endl;
return 0;
}

File: week4/student.java
import java.util.Scanner; 
class Student010
{
 public String fullName; 
int rollNum; 
double semPercentage;
String collegeName;
 int collegeCode; 
Student010()
{ 
     System.out.println("Student class is created successfully!!\nIam a student\n");
 } 
 protected void finalize() throws Throwable
{ 
   System.out.println("The class is finalized\n"); 
} 
}
public class Student
{
 public static void main(String[] args) 
{ 
Student010 priya=new Student010();; 
Scanner input=new Scanner(System.in); 
System.out.print("Enter your sweet name : "); 
priya.fullName=input.nextLine(); 
System.out.print("Enter your rollNumber : "); 
priya.rollNum=input.nextInt(); 
System.out.print("Enter your semPercentage : "); priya.semPercentage=input.nextDouble(); 
input.nextLine(); 
System.out.print("Enter your collegeName : "); 
priya.collegeName=input.nextLine(); 
System.out.print("Enter your collegeCode : "); 
priya.collegeCode=input.nextInt(); 
System.out.print("Name : "+priya.fullName+"\n"); 
System.out.print("RollNumber : "+priya.rollNum+"\n"); System.out.print("SemPercentage : "+priya.semPercentage+"\n"); System.out.print("CollegeName : "+priya.collegeName+"\n"); System.out.print("CollegeCode : "+priya.collegeCode+"\n"); input.close();
 } 
} 
